src/lib/components/colors/Header.svelte

```diff
@@ -171,6 +179,18 @@
     grid-area: swatch;
     height: 100%;
     position: relative;
+    &.out-of-gamut {
+      outline: var(--warning) 3pt solid;
+      outline-offset: 1pt;
+    }
+
+    .gamut-warning {
+      --icon-color: var(--warning);
```

Comment:
It seems like we should make sure this always has contrast. I wonder about leaving the text visible (remove sr-only above, and move text after the icon) then styling it as white on a warning-color background, with a white outline? Does that description make sense?

- The white outline always gives it an offset from the chosen color
- The warning background gives it a consistent warning color
- The white text is guaranteed to have contrast on the warning background

Would something like that make sense?

Reply:
Yeah, that makes total sense. Cleaned this up in https://github.com/oddbird/oddcontrast/pull/226/commits/c4c41d1c77528080b331fe9f9e45d6a94600c53a

![image](https://github.com/user-attachments/assets/0b46208c-d50b-4898-92c9-3016aca4c44b)